<html>
<head>
<title>TestNG:  Default test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Default test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>17/2/0</td>
</tr><tr>
<td>Started on:</td><td>Fri Jun 07 16:04:50 IST 2024</td>
</tr>
<tr><td>Total time:</td><td>29 seconds (29884 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='newweb.vectorfile2.cancelUnshippedOrder()'><b>cancelUnshippedOrder</b><br>Test class: newweb.vectorfile2</td>
<td><div><pre>java.lang.AssertionError: There should be at least one unshipped order expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertFalse(Assert.java:67)
	at newweb.vectorfile2.cancelUnshippedOrder(vectorfile2.java:702)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div><a href='#' onClick='toggleBox("stack-trace1031479068", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1031479068'><pre>java.lang.AssertionError: There should be at least one unshipped order expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertFalse(Assert.java:67)
	at newweb.vectorfile2.cancelUnshippedOrder(vectorfile2.java:702)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.reviewReturnConfirmationEmailContent()'><b>reviewReturnConfirmationEmailContent</b><br>Test class: newweb.vectorfile2</td>
<td><div><pre>java.lang.AssertionError: The email should contain detailed next steps for the return process expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertTrue(Assert.java:45)
	at newweb.vectorfile2.reviewReturnConfirmationEmailContent(vectorfile2.java:571)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div><a href='#' onClick='toggleBox("stack-trace174871747", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace174871747'><pre>java.lang.AssertionError: The email should contain detailed next steps for the return process expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:99)
	at org.testng.Assert.failNotEquals(Assert.java:1037)
	at org.testng.Assert.assertTrue(Assert.java:45)
	at newweb.vectorfile2.reviewReturnConfirmationEmailContent(vectorfile2.java:571)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='newweb.vectorfile2.waitForOrderToBeOutForDelivery()'><b>waitForOrderToBeOutForDelivery</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.checkReturnRequestConfirmationEmail()'><b>checkReturnRequestConfirmationEmail</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.navigateToHeaderSection()'><b>navigateToHeaderSection</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.placeOrderAndReceiveConfirmationEmail()'><b>placeOrderAndReceiveConfirmationEmail</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.locateHelpCenterLink()'><b>locateHelpCenterLink</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.initiateReturnRequest()'><b>initiateReturnRequest</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.receiveConfirmationEmailForCancellations()'><b>receiveConfirmationEmailForCancellations</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.verifyHelpCenterContent()'><b>verifyHelpCenterContent</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.checkHelpCenterLinkInFooter()'><b>checkHelpCenterLinkInFooter</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.waitForOrderToBeProcessedAndShipped()'><b>waitForOrderToBeProcessedAndShipped</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.locateFooterSection()'><b>locateFooterSection</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.navigateToOrderHistory()'><b>navigateToOrderHistory</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.accessHelpCenterFromAnyPage()'><b>accessHelpCenterFromAnyPage</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.checkHelpCenterLinkInHeader()'><b>checkHelpCenterLinkInHeader</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.accessHelpCenterPage()'><b>accessHelpCenterPage</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.receiveConfirmationEmailForReturns()'><b>receiveConfirmationEmailForReturns</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
<tr>
<td title='newweb.vectorfile2.verifyHelpCenterContentSections()'><b>verifyHelpCenterContentSections</b><br>Test class: newweb.vectorfile2</td>
<td></td>
<td>0</td>
<td>newweb.vectorfile2@242b836</td></tr>
</table><p>
</body>
</html>